src/components/Welcome.jsx

```diff
@@ -1,13 +1,30 @@
+"use client";
+
+import React, { useEffect } from "react";
 import Image from "next/image";
 import Waves from "@/public/svg/landingWaves.svg";
 import { FaChevronDown } from "react-icons/fa";
 import Navigation from "@/components/Navigation";
 
 const Welcome = () => {
+  useEffect(() => {
```

Comment:
should be able to do this without a use effect, checkout designverse navigation system